src/test/java/com/dreamsportslabs/guardian/it/PasswordlessInitIT.java

```diff
@@ -0,0 +1,1414 @@
+import static com.dreamsportslabs.guardian.Constants.BODY_PARAM_TEMPLATE;
+import static com.dreamsportslabs.guardian.Constants.BODY_PARAM_USERID;
+import static com.dreamsportslabs.guardian.Constants.BODY_PARAM_USERNAME;
+import static com.dreamsportslabs.guardian.Constants.CODE;
+import static com.dreamsportslabs.guardian.Constants.DB_CREATED_AT_EPOCH;
+import static com.dreamsportslabs.guardian.Constants.DB_EXPIRY;
+import static com.dreamsportslabs.guardian.Constants.DB_IS_OTP_MOCKED;
+import static com.dreamsportslabs.guardian.Constants.ERROR;
+import static com.dreamsportslabs.guardian.Constants.ERROR_INVALID_REQUEST;
+import static com.dreamsportslabs.guardian.Constants.ERROR_INVALID_STATE;
+import static com.dreamsportslabs.guardian.Constants.ERROR_RESENDS_EXHAUSTED;
+import static com.dreamsportslabs.guardian.Constants.ERROR_RESENDS_NOT_ALLOWED;
+import static com.dreamsportslabs.guardian.Constants.ERROR_RETRIES_EXHAUSTED;
+import static com.dreamsportslabs.guardian.Constants.ERROR_USER_EXISTS;
+import static com.dreamsportslabs.guardian.Constants.ERROR_USER_NOT_EXISTS;
+import static com.dreamsportslabs.guardian.Constants.MESSAGE;
+import static com.dreamsportslabs.guardian.Constants.METADATA;
+import static com.dreamsportslabs.guardian.Constants.RESPONSE_BODY_PARAM_TRIES;
+import static com.dreamsportslabs.guardian.utils.ApplicationIoUtils.passwordlessInit;
+import static com.dreamsportslabs.guardian.utils.DbUtils.insertState;
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathMatching;
+import static org.apache.http.HttpHeaders.CONTENT_TYPE;
+import static org.apache.http.HttpStatus.SC_BAD_REQUEST;
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.both;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.greaterThanOrEqualTo;
+import static org.hamcrest.Matchers.isA;
+import static org.hamcrest.Matchers.lessThanOrEqualTo;
+
+import com.dreamsportslabs.guardian.utils.DbUtils;
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.stubbing.StubMapping;
+import io.restassured.response.ValidatableResponse;
+import io.vertx.core.json.JsonObject;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.hamcrest.core.CombinableMatcher;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class PasswordlessInitIT {
+  public static String tenantId_1 = "tenant1"; // OTP is mocked for this tenant
+  public static String tenantId_2 = "tenant2"; // OTP is not mocked for this tenant
```

Comment:
```suggestion
  public static String tenant1 = "tenant1"; // OTP is mocked for this tenant
  public static String tenant2 = "tenant2"; // OTP is not mocked for this tenant
```
Avoid using _ where camel casing convention followed.